src/hooks/useShellEnv.ts

```diff
@@ -66,13 +65,6 @@ export default async function useShellEnv({installations}: Options): Promise<Rec
       vars.ACLOCAL_PATH = compact_add(vars.ACLOCAL_PATH, installation.path.join("share/aclocal").compact()?.string)
     }
 
-    if (installation.pkg.project === 'openssl.org') {
-      const certPath = installation.path.join("ssl/cert.pem").compact()?.string
```

Comment:
Fortunately this was already defensively programmed. So we’re safe! Path.compact is undefined if the file doesn't exist.

Reply:
Ah, then that will work, yes. As long as it doesn't overwrite runtime.env.SSL_CERT_FILE value from curl.se/ca-certs with nothing, all is well.